pulsar-zookeeper-utils/src/main/java/com/yahoo/pulsar/zookeeper/ZooKeeperCache.java

```diff
@@ -89,7 +94,7 @@ public ZooKeeperCache(ZooKeeper zkSession, OrderedSafeExecutor executor) {
     }
 
     public ZooKeeperCache(ZooKeeper zkSession) {
-        this(zkSession, new OrderedSafeExecutor(1, "zk-cache-executor"));
+        this(zkSession, new OrderedSafeExecutor(1, "zk-cache-executor"), Executors.newSingleThreadScheduledExecutor());
```

Comment:
Also, we should give a name the new executors threads, for easier understanding of jstack traces